lib/tbd_tests.rb

```diff
@@ -27,28 +27,3 @@
 # CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
 # OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-
```

Comment:
**TBD Tests** may have been initially envisioned as a Ruby gem (way back when). Not the case - will remain a larger repository of tests, pulling _TBD_ as a gem/measure. Keeping as a placeholder if ever the decision to _gemify_ were reversed.